engine.py

```diff
@@ -186,15 +188,18 @@ def make_move(self, move):
 		"""
 		self.board[move.start_row][move.start_col] = "  "
 		self.board[move.end_row][move.end_col] = move.piece_moved
-		self.move_log.append(move) # log move
-		self.light_to_move = not self.light_to_move # next player to move
+		#######
 		##
 		#Updating the postion of the Kings
 		##
-		if move.piece_moved == "kl": #  light king pices moved
-			self.light_king_pos = (move.end_row, move.end_col) # upadate tuple containging postion
-		elif move.piece_moved == "kd": # Dark king pices moved
-			self.dark_king_pos = (move.end_row, move.end_col)# upadate tuple containging postion
+		if move.piece_moved == "kl": #  light king piece moved
+			self.light_king_pos = (move.end_row, move.end_col) # upadate tuple containing postion
+		elif move.piece_moved == "kd": # Dark king piece moved
+			self.dark_king_pos = (move.end_row, move.end_col)# upadate tuple containing postion
```

Comment:
if you make the changes suggested above, here you can just do

if move.piece_moved[0] = "k":
    self.king_pos[piece_moved] = (move.end_row, move.end_col)

And there will be no need to have separate lines for dark and light king

Reply:
good suggestions... changes write away